modules/end-user-guide/pages/customizing-workspaces-components.adoc

```diff
@@ -1,12 +1,16 @@
-:navtitle: Customizing workspaces components
+:navtitle: Customizing workspace components
```

Comment:
Delete the old page when you are renaming it, add an alias for the old page name, and update all xrefs.